b-locker-web/src/components/organisation/topbar/orgTopbar.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './orgTopbar.scss';
+
+import navUserIcon from '../../../assets/userTopbar.svg'
+
+const topbar: React.FC = () => {
+
+    return (
+        <section className="topbar">
+            <div className="container">
+                <a><img className="user-icon" src={navUserIcon} alt='' />Brian </a>
+                
+                <a href="#news">Admin </a>
```

Comment:
change <a> tags to <button> tags to prevent having invalid href's.

Reply:
bij het gebruik van <button> ipv <a> last van user agent stylesheet met default css waarden voor buttons

Reply:
Ik kijk morgen wel ff mee